ostelco-ios-client/ViewControllers/AuthParentViewController.swift

```diff
@@ -29,6 +30,15 @@ class AuthParentViewController: UIViewController, OnboardingCoordinatorDelegate
         
         let tabs = UIStoryboard(name: "TabController", bundle: nil).instantiateInitialViewController()
         embedFullViewChild(tabs!)
+        
+        let controller = LocationController.shared
+        if controller.authorizationStatus == .notDetermined {
```

Comment:
What happens if `authorizationStatus == .denied`?

Reply:
If they refuse, then we can't try to be helpful, right?

Reply:
I was under the impression that we would block the app if permission is denied, so they won't see the home screen. Exactly the same functionality for location permissions we have in AllowLocationPermissionsVC that we used in the onboarding flow (after user has logged in), move that logic out of the onboarding, but make sure it's handled all over the app

Reply:
You're right, I confirmed. Fixed now.